build/dockerfiles/rhel.Dockerfile

```diff
@@ -52,6 +52,16 @@ RUN ./index.sh > /build/devfiles/index.json
 RUN ./list_referenced_images.sh devfiles > /build/devfiles/external_images.txt
 RUN chmod -R g+rwX /build/devfiles
 
+
+FROM registry.access.redhat.com/ubi8/nodejs-12:1-102 as dwtemplates
```

Comment:
probably should use a more recent version (12 is no longer an active LTS version)

Reply:
Changed to `registry.access.redhat.com/ubi8/nodejs-14-minimal:1-31`

Reply:
why not nodejs-16, 14 is no longer active as well ?
https://nodejs.org/en/about/releases/#releases

Reply:
(also you pickup nodejs lts for alpine which is for now v16.13.0.)